tests/System.Azure.Experimental.Tests/AuthenticationTests.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Azure.Authentication;
+using System.Buffers.Cryptography;
+using System.Text;
+using Xunit;
+
+namespace tests
```

Comment:
nit: rename namespace to be consistent with other test projects? System.Azure.Tests?